index.js

```diff
@@ -7,7 +7,9 @@ const XXH = require('xxhashjs');
 
 const H = XXH.h32(0xABCD) // seed = 0xABCD
 
-const appIndexJs = path.resolve(pwd, 'src/index.js');
+const appIndexes = ['js', 'ts', 'jsx', 'tsx'].map(
+  ext => path.resolve(pwd, `src/index.${ext}`
+);
```

Comment:
Missing right parentheses;

Reply:
was already fixed in https://github.com/Derek-Hu/react-app-rewire-multiple-entry/pull/4/commits/ad7ddb6096050787c28233179999bcb0f2f0a2f1